### DISCUSSION TITLE
What is periodic scanner behavior for files that are being uploaded

### DISCUSSION THREAD

**Discussion by mawi675lr81** [photoview/photoview#598] [Q&A and Troubleshooting]
If a file was being uploaded to nextcloud directory and at the same time periodic scanner gets triggered, how does photoview handle it?

**Comment by mawi675lr81** [comment]
Anyone?

**Comment by viktorstrate** [comment]
I think it depends on the software that uploads it, some software adds a `.part` extension to files while they are being uploaded and then renames them when they are fully uploaded. In that case the scanner will ignore it and grab them the next time it runs, and they are fully uploaded.

If the uploading software does not add `.part` or similar, then in the worst case the scanner will detect it and try to generate thumbnails for it which will fail, an error message will be shown on the website and next time the scanner runs it will retry.

tl;dr it shouldn't be a problem
